package.json

```diff
@@ -17,10 +17,12 @@
     "@react-native-community/geolocation": "^2.0.2",
     "@react-native-community/masked-view": "^0.1.7",
     "@react-native-community/toolbar-android": "0.1.0-rc.2",
-    "@react-native-mapbox-gl/maps": "^8.0.0",
+    "@react-native-mapbox-gl/maps": "^8.1.0",
```

Comment:
The iOS archive process fails using this version, see https://github.com/react-native-mapbox-gl/maps/issues/1097

Reply:
@ankitecd is there any special reason for using the new version 8.1.0 or can I downgrade to 8.0.0 again?

Reply:
I think I made the beta work.

Reply:
> @ankitecd is there any special reason for using the new version 8.1.0 or can I downgrade to 8.0.0 again?

You can downgrade if required

Reply:
Beta is working.